geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateIndexCommand.java

```diff
@@ -64,26 +65,40 @@ public Result createIndex(@CliOption(key = CliStrings.CREATE_INDEX__NAME, mandat
 
       @CliOption(key = {CliStrings.GROUP, CliStrings.GROUPS},
           optionContext = ConverterHint.MEMBERGROUP,
-          help = CliStrings.CREATE_INDEX__GROUP__HELP) final String[] group) {
+          help = CliStrings.CREATE_INDEX__GROUP__HELP) final String[] groups) {
 
-    Result result;
-    final Set<DistributedMember> targetMembers = findMembers(group, memberNameOrID);
+    CommandResult result;
+    final Set<DistributedMember> targetMembers = findMembers(groups, memberNameOrID);
 
     if (targetMembers.isEmpty()) {
       return ResultBuilder.createUserErrorResult(CliStrings.NO_MEMBERS_FOUND_MESSAGE);
     }
 
-    IndexInfo indexInfo = new IndexInfo(indexName, indexedExpression, regionPath, indexType);
+    RegionConfig.Index index = new RegionConfig.Index();
+    index.setName(indexName);
+    index.setExpression(indexedExpression);
+    index.setFromClause(regionPath);
```

Comment:
The `FROM` clause is not necessarily the `regionPath`.  A region `/exampleRegion` can have the clause `FROM /exampleRegion.keySet`.

[Docs](https://geode.apache.org/docs/guide/13/developing/query_select/the_from_clause.html#the_from_clause).

Since the config object doesn't contain the region path explicitly, it will need to be sent along with the config object in the `executeAndGetFunctionResults` below.  This will be a common pattern for many commands and could warrant its own wrapper a la

```
    class RegionConfigCommandWrapper<T extends CacheElement> implements Serializable {
      private T configObject;
      private String region;
      private Object[] additionalParameters;

      public RegionConfigCommandWrapper(T configObject, String region,
          Object[] additionalParameters) {
        this.configObject = configObject;
        this.region = region;
        this.additionalParameters = additionalParameters;
      }
    }
```